plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -519,4 +521,30 @@ public IcebergConfig setMetadataCacheEnabled(boolean metadataCacheEnabled)
         this.metadataCacheEnabled = metadataCacheEnabled;
         return this;
     }
+
+    public boolean isObjectStoreEnabled()
+    {
+        return objectStoreEnabled;
+    }
+
+    @Config("iceberg.object-store.enabled")
+    @ConfigDescription("Enable the Iceberg object store file layout")
+    public IcebergConfig setObjectStoreEnabled(boolean objectStoreEnabled)
+    {
+        this.objectStoreEnabled = objectStoreEnabled;
+        return this;
+    }
+
+    public Optional<String> getDataLocation()
+    {
+        return dataLocation;
+    }
+
+    @Config("iceberg.data-location")
+    @ConfigDescription("Path for data files")
+    public IcebergConfig setDataLocation(String dataLocation)
```

Comment:
Is there any situation when we want to set only `iceberg.data-location` without enabling `iceberg.object-store.enabled`?

Reply:
The data files go directly into the location specified with `iceberg.data-location`.

For example `local:///table-location/xyz/20241113_190252_00001_a893a-0381cc12-3db2-4e3b-981e-42b52ee47e2e.parquet`

Reply:
It rather increases the possibility of throttling, right? Please consider disallowing such combination if it's not a common scenario.